features/rules/OJP/OJP000_Object-placement.feature

```diff
@@ -0,0 +1,13 @@
+@implementer-agreement
+@OJP
+@version1
+Feature: OJP000 - Object placement
+The rule verifies the presence of an object placement to define the overall local coordinate system and placement of the product in space
```

Comment:
@Ghesselink please review the descriptions of all 000 rules. They must always, consistently start with "The rule verifies the presence of IFC entities used to..."
cc: @civilx64